misc/ls-extensions/modules/json-to-record-converter/src/main/java/io/ballerina/converters/JsonToRecordConverter.java

```diff
@@ -454,7 +454,11 @@ private static OpenAPI parseJSONSchema(String schemaString, String recordName) t
             cleanedMap.replace("properties", properties);
         } else if (cleanedMap.get("type").equals("array")) {
             Map<String, Object> itemsSchema = (Map<String, Object>) cleanedMap.get("items");
-            cleanedMap.replace("items", cleanSchema(itemsSchema));
+            if (itemsSchema != null && itemsSchema.size() != 0) {
```

Comment:
shall we use `Map.isEmpty()` to check the empty map? Also, maybe we can simplify the replace method usage as well. Try to carry out the cleanup logic within the if clause so we can remove the else clause and have one replace method usage